com.hahoto.onelimx.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.hahoto.onelimx
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: 1LimX
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+modules:
+  - name: 1LimX
```

Comment:
Nothing is getting moved or installed anywhere. Did you build and test it locally?

Reply:
Thanks! Let me review it again.